club/theme.json

```diff
@@ -137,7 +141,7 @@
                 },
                 {
                     "name": "Extra Large",
-                    "size": "2rem",
+                    "size": "2.5rem",
```

Comment:
Ah I think I understand the answer to my question above — the Figma specifies an Extra Large font size in the paragraph font sizes, but it is not responsive. But in the page layouts, this same font size is used in many places at desktop widths, but scales down on mobile.

I wonder if we can consolidate them and have the extra large preset use the responsive value (i.e. 40px > 20px)

Reply:
Right. It would be simpler to use the preset by making it a variable size. 
But making extra large a variable size between `40 -> 20px` leaves large at `28px`. Also I am not sure if `20px` lower value is good idea because medium is already at 20.

@beafialho please suggest if it makes sense to make typography to be variable sizes.

Or probably we could rename the responsive variable to be more meaningful.

Reply:
renamed the custom variable to `default-title`